providers/pinpoint/src/lib/pinpoint.provider.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { PinpointSMSVoiceV2 } from '@aws-sdk/client-pinpoint-sms-voice-v2';
+import { PinpointSMSProvider } from './pinpoint.provider';
+
+test('should trigger pinpoint library correctly', async () => {
+  const mockResponse = { MessageId: 'mock-message-id' };
+  const spy = jest
+    .spyOn(PinpointSMSVoiceV2.prototype, 'send')
+    .mockImplementation(async () => {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      return mockResponse as any;
+    });
+
+  const mockConfig = {
+    region: 'test-1',
+    accessKeyId: 'TEST',
+    from: '+112345',
+    secretAccessKey: 'TEST',
+  };
+  const provider = new PinpointSMSProvider(mockConfig);
+
+  const mockNovuMessage = {
+    content: 'Hello from Novu!',
+    to: '+1234567890',
+    from: mockConfig.from,
+  };
+  const response = await provider.sendMessage(mockNovuMessage);
+
+  expect(spy).toHaveBeenCalled();
```

Comment:
We should check here with which payload the spy has been called.

Reply:
As I said in the issue I have no in-depth knowledge of unit testing especially unit testing libraries. I would need some help with this step.

Reply:
I see. But when I try to execute the tests I get an error that region `test-1` is not valid, so I switched to `eu-central-1`. Now I get an error that my security token is invalid. I'm not really fond of putting one of my tokens in there.

Reply:
And you shouldn't put your token, that's right. But it sounds that the provider client is not properly mocked and it is trying to call the service. Let's run the CI and see what is returning so we might find help for you.